lib/functional_mixin.js

```diff
@@ -95,6 +108,58 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.swipeLeft = function busterFunctionalMixin_swipeLeft(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
```

Comment:
so for swipeLeft and right, the window width is used which is multiple by a factor of 0.75, coz usually when you swipe horizontally its usually like 75% of the screen width. If window is not defined just use a default swipeRight left width? Is there a better way to do this ??